src/app/external-cluster-wizard/steps/external-cluster/provider/aks/component.ts

```diff
@@ -169,6 +171,7 @@ export class AKSClusterSettingsComponent
         Validators.min(this.AUTOSCALING_MIN_VALUE),
       ]),
     });
+    this._getAKSKubernetesVersions();
```

Comment:
move this method to `initSubscriptions`
Here we handle ALL
- form control subscriptions
- side effect calls

Reply:
but i need to be called just one time when component shown